### PR TITLE
Added the datadog and rollbar recipes

### PR DESCRIPTION
**Pull request by TheOriginalAlex**
Signed-off-by: Alex Johnson <ajohnson@bombora.com>

### CODE REVIEW

File: recipes/datadog/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - simplejson >=2.0.9
+
+test:
+  imports:
+  - datadog
+  - datadog.dogshell
+
+about:
+  home: https://www.datadoghq.com/
+  license: BSD
```
Comment (CJ-Wright) [inline recipes/datadog/meta.yaml]: Which BSD?
Please include a license_file and a license_family

File: recipes/datadog/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  summary: 'The Datadog Python library'
+
+  description: |
+    The Python integration enables you to monitor any custom metric by instrumenting a few lines of code.
+  doc_url: https://docs.datadoghq.com/integrations/python/
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```
Comment (CJ-Wright) [inline recipes/datadog/meta.yaml]: Please remove example comments

File: recipes/rollbar/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+test:
+  requires:
+    - unittest2
+  imports:
+  - rollbar
+  - rollbar.test
+
+about:
+  home: https://github.com/rollbar/pyrollbar
+  license: MIT
```
Comment (CJ-Wright) [inline recipes/rollbar/meta.yaml]: See applicable comments from above.

File: recipes/datadog/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  run:
+    - python
+    - decorator >=3.3.2
+    - requests >=2.6.0
+    - simplejson >=2.0.9
+
+test:
+  imports:
+  - datadog
+  - datadog.dogshell
```
Comment (CJ-Wright) [inline recipes/datadog/meta.yaml]: You might want to test the cli (if only to check that it exists)

**Comment by TheOriginalAlex** [comment]
@CJ-Wright: Fixed!

**Comment by CJ-Wright** [comment]
Great!
Welcome to Conda-Forge.

**Comment by CJ-Wright** [comment]
@TheOriginalAlex Just for the future please have one recipe per PR.

**Pull request merged**